test/integration/render-tests/regressions/mapbox-gl-js#5631/style.json

```diff
@@ -0,0 +1,74 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 64,
+      "height": 64
+    }
+  },
+  "sources": {
+    "geojson": {
+      "type": "geojson",
+      "data": {
+        "type": "FeatureCollection",
+        "features": [
+          {
+            "type": "Feature",
+            "properties": {
+              "name": "A",
+              "size": 12
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                -10,
+                0
+              ]
+            }
+          },
+          {
+            "type": "Feature",
+            "properties": {
+              "name": "B",
+              "size": 24
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                10,
+                0
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "sprite": "local://sprites/icon-text-fit",
+  "glyphs": "local://glyphs/{fontstack}/{range}.pbf",
+  "layers": [
+    {
+      "id": "symbol",
+      "type": "symbol",
+      "source": "geojson",
+      "layout": {
+        "text-font": [
+          "Open Sans Semibold",
+          "Arial Unicode MS Bold"
+        ],
+        "text-field": {
+          "type": "identity",
+          "property": "name"
+        },
+        "text-size": {
+          "type": "identity",
+          "property": "size"
+        },
+        "icon-image": "label",
+        "icon-text-fit": "both",
+        "icon-ignore-placement": true,
```

Comment:
I'm not sure why `icon-ignore-placement` and `icon-allow-overlap` were needed here, because the two symbols don't appear to collide. But without them, only "A" shows up. Might be another bug lurking here.

Reply:
Yeah, it looks like there is a bug -- the icons have collision boxes that are much bigger than the text:

![screenshot 2017-11-08 17 27 30](https://user-images.githubusercontent.com/375121/32583738-1e900616-c4aa-11e7-880c-da6f16f7a468.png)

I'll look into that tomorrow.